docs/src/modules/components/Link.js

```diff
@@ -29,31 +28,49 @@ function Link(props) {
   const {
     activeClassName = 'active',
     className: classNameProps,
+    href: routerHref,
     innerRef,
     naked,
     role: roleProp,
     ...other
   } = props;
+
+  // apply nextjs rewrites
+  const href = routerHref.replace(/\/api-docs\/(.*)/, '/api/$1');
+
   const router = useRouter();
 
   const userLanguage = useSelector(state => state.options.userLanguage);
   const className = clsx(classNameProps, {
-    [activeClassName]: router.pathname === props.href && activeClassName,
+    [activeClassName]: router.pathname === routerHref && activeClassName,
   });
 
-  if (userLanguage !== 'en' && other.href.indexOf('/') === 0 && other.href.indexOf('/blog') !== 0) {
-    other.as = `/${userLanguage}${other.href}`;
+  if (userLanguage !== 'en' && href.indexOf('/') === 0 && href.indexOf('/blog') !== 0) {
+    other.as = `/${userLanguage}${href}`;
   }
 
   // catch role passed from ButtonBase. This is definitely a link
   const role = roleProp === 'button' ? undefined : roleProp;
 
+  const isExternal = href.startsWith('https:') || href.startsWith('mailto:');
```

Comment:
IE 11

Reply:
@oliviertassinari Do you still observe this? If so could you describe how you got there and post a screenshot? Can't identify issues on i11 and win 10

Reply:
Looking at it

Reply:
All good